src/TodoList.js

```diff
@@ -1,27 +1,27 @@
 import React from "react";
-export default TodoList;
+import TodoListItem from "./TodoListItem";
 
-const todoList = [
-  {
-    id: 1,
-    title: "code",
-  },
-  {
-    id: 2,
-    title: "go to store",
-  },
-  {
-    id: 3,
-    title: "attend group session",
-  },
-];
+export default function TodoList() {
+  const todoList = [
+    {
+      id: 1,
+      title: "code",
+    },
+    {
+      id: 2,
+      title: "go to store",
+    },
+    {
+      id: 3,
+      title: "attend group session",
+    },
+  ];
 
-function TodoList() {
   return (
     <div>
       <ul>
         {todoList.map(function (item) {
-          return <li key={item.id}>{item.title}</li>;
+          return <TodoListItem key={todoList.id} todo={item.title} />;
```

Comment:
need to change this to `item.id` so each `TodoListItem` will receive a unique Id:

```suggestion
          return <TodoListItem key={item.id} todo={item.title} />;
```